scripts/README.md

```diff
@@ -5,36 +5,41 @@
 The respective shell scripts contained within this directory can be utilised to generate application specific compliant
 configuration which can be utilised with:
 
-* Dnsmasq
-* Unbound
 * AdGuard Home
+* BIND9
+* Dnsmasq/Pi-hole
+* Squid
+* Unbound
 
 ## Usage
 
 1. Copy `config.example.json` to `config.json`.
 2. Modify `config.json` to include your Cacheserver's IP(s) and the CDNs you plan to cache.
-   The following example assumes a single shared Cacheserver IP:
+
+The following example assumes a single shared Cacheserver IP:
 ```json
 {
-  "ips": {
-    "generic":	["10.10.10.200"]
-  },
-  "cache_domains": {
-    "blizzard":     "generic",
-    "epicgames":    "generic",
-    "nintendo":     "generic",
-    "origin":       "generic",
-    "riot":         "generic",
-    "sony":         "generic",
-    "steam":        "generic",
-    "uplay":        "generic",
-    "wsus":         "generic"
-  }
+    "combined_output": false,
+    "ips": {
+        "generic":      ["10.10.10.200"]
```

Comment:
Should this reference monolithic instead of generic?

Reply:
Adjusted.